hack/update-codegen.sh

```diff
@@ -18,7 +18,8 @@ trap "cleanup" EXIT SIGINT
 bash "${CODEGEN_PKG}/generate-groups.sh" all \
   "${OPERATOR_PACKAGE_ROOT}/pkg/generated" "${OPERATOR_PACKAGE_ROOT}/pkg/apis" \
   "acid.zalan.do:v1 zalando.org:v1" \
-  --go-header-file "${SCRIPT_ROOT}"/hack/custom-boilerplate.go.txt
+  --go-header-file "${SCRIPT_ROOT}"/hack/custom-boilerplate.go.txt \
+  -o ./
```

Comment:
unrelated change. What is it for?

Reply:
This is fix when GOPATH installed explicitly (for example in GoLand IDE and multiple golang version installed) code generation  will be targeted to the GOPATH\src instead of the root project path.